Source/PropertyTools.Wpf/ItemsBag/ItemsBagPropertyDescriptor.cs

```diff
@@ -157,7 +157,11 @@ public override void ResetValue(object component)
         public override void SetValue(object component, object value)
         {
             var bag = (ItemsBag)component;
-            bag.SuspendNotifications = true;

```

Comment:
instead of commenting out and adding a comment, I suggest to remove the line completely. Is "SuspendNotifications" not needed any more?

Reply:
We do not use SuspendNotifications anymore. The only place we use this is in the ItemsBag context.